### ISSUE TITLE
option `--paper` ignored

### ISSUE THREAD

**Issue by btwe** [rrthomas/pdfjam#82]
On archlinux:

```
% pdfjam --version
3.10
```

The options `--paper A4` and `--a4paper` are ignored. The commands produces PDF files with page size `letter`. (Which seem to correspond to my locale, but is not the desired result):

```
% pdfjam --quiet --outfile out.pdf --paper A4 -- in.png && pdfinfo out.pdf | grep '^Page size'
Page size:       612 x 792 pts (letter)

% pdfjam --quiet --outfile out.pdf --a4paper -- in.png && pdfinfo out.pdf | grep '^Page size'
Page size:       612 x 792 pts (letter)
```
IMO, command line options should have highest priority and not be overridden by locale or environment settings. That is why I file this issue.

A workaround was found setting the environ `PAPERSIZE`:
```
% PAPERSIZE=A4 pdfjam --quiet --outfile out.pdf -- in.png && pdfinfo out.pdf | grep '^Page size'
Page size:       595.276 x 841.89 pts (A4)
```

TIA

**Comment by lemniscati** [comment]
Try [v3.12](https://github.com/rrthomas/pdfjam/releases/tag/v3.12), please.

**Comment by btwe** [comment]
> Try [v3.12](https://github.com/rrthomas/pdfjam/releases/tag/v3.12), please.
Thanks for the quick feedback. I'll wait until archlinux provides the new version. They ship pdfjam in their `texlive-binextra` package. I'll comment here later.

**Comment by akhuettel** [comment]
I think this is fixed in v3.12.

**Comment by btwe** [comment]
I can confirm that the functionality of the `--a4paper` option in version `v3.12` is now working as expected. I'm sorry for letting you wait. Testing the new release turned out to be less complex than initially thought.

The archlinux package still ships with `3.10` though. I try to find the correct place to trigger an update there as well.

Many thanks,
